### PR TITLE
Object segmentation example

### PR DESCRIPTION
**Pull request by gmalinve**

### CODE REVIEW

File: examples/03-Maxwell/Maxwell3D_Segmentation.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+magnet = [obj for obj in mod3D.object_list if obj.name == object_name][0]
+sheets_3 = mod3D.objects_segmentation(magnet, segmentation_thickness=segmentation_thickness, apply_mesh_sheets=True)
+
+##################################################################################
+# Segment fourth magnet by specifying the number of segments
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Select fourth magnet to segment by specifying the number of segments.
+# In this specific case we give as input the name of the magnet and we disable the mesh sheets.
+
+object_name = "PM_O1_1"
```
Comment (MaxJPRey) [inline examples/03-Maxwell/Maxwell3D_Segmentation.py:91]: @gmalinve not sure you are using `object_name` here and providing the name of the magnet.

File: examples/03-Maxwell/Maxwell3D_Segmentation.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                specified_version="2023.1",
+                new_desktop_session=True,
+                non_graphical=non_graphical)
+
+##################################################################################
+# Create object to access 3D modeler
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create the object ``mod3D`` to access the 3D modeler easily.
+
+mod3D = m3d.modeler
```
Comment (MaxJPRey) [inline examples/03-Maxwell/Maxwell3D_Segmentation.py:47]: I would probably name the variable modeler here.
```suggestion
modeler = m3d.modeler
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: examples/03-Maxwell/Maxwell3D_Segmentation.py (hunk 1 above)
Comment (gmalinve) [reply examples/03-Maxwell/Maxwell3D_Segmentation.py:91]: you are right! :) thanks!

**Pull request merged**